### PR TITLE
Added tests for comment and about us applications

### PR DESCRIPTION
**Pull request by Rysbai**

### CODE REVIEW

File: aboutus/tests.py [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+        model = AboutUs
+
+    payload_kg = 'Example payload in kyrgyz'
+    payload_ru = 'Example payload in russian'
+
+
+class AboutUsModelTest(TestCase):
+    def test_should_raise_validate_error_if_aboutus_already_exist(self):
+        first = AboutUs.objects.create(payload_kg='First about us', payload_ru='First about us')
+        second = AboutUs.objects.create(payload_kg='Second about us', payload_ru='Second about us')
```
Comment (dadaday) [inline aboutus/tests.py:17]: Have you tried to use `AboutUsFactory` for this test?

File: aboutus/tests.py [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+            second.save()
+
+
+class AboutUsAPITest(TestCase):
+    def test_should_return_about_us_content_in_kyrgyz_by_default(self):
+        aboutus = AboutUsFactory()
+        path = '/api/aboutus/'
+
+        response = self.client.get(path)
+        data = json.loads(response.content.decode())
```
Comment (dadaday) [inline aboutus/tests.py]: Give a try to use `data = response.json()` and see if it works

File: aboutus/tests.py [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+        first.save()
+
+        with self.assertRaises(ValidationError):
+            second.save()
+
+
+class AboutUsAPITest(TestCase):
+    def test_should_return_about_us_content_in_kyrgyz_by_default(self):
+        aboutus = AboutUsFactory()
+        path = '/api/aboutus/'
```
Comment (dadaday) [inline aboutus/tests.py]: You can use `reverse` for getting paths

File: aboutus/tests.py [hunk 4]
```diff
@@ -1,3 +1,47 @@
...
+from django.core.exceptions import ValidationError
+from rest_framework import status
 from django.test import TestCase
+from factory import DjangoModelFactory
 
-# Create your tests here.
+from aboutus.models import AboutUs
+
+
+class AboutUsFactory(DjangoModelFactory):
```
Comment (dadaday) [inline aboutus/tests.py]: You can put Factories in separate files

File: comment/tests.py [hunk 5]
```diff
@@ -1,3 +1,68 @@
...
+        self.assertEqual(body['text'], data['text'])
+
+    def test_should_send_mail_to_admin(self):
+        path = '/api/comments/'
+        data = {
+            'name': 'Example name',
+            'phone': '+996779583738',
+            'text': 'Some comments here'
+        }
+        utils.send_comment_to_admin_email = mock.Mock(return_value=None)
```
Comment (dadaday) [inline comment/tests.py]: nice usage of mock! 👍
Also there is an option to use decorators for mocking (have a look at them)

Comment (dadaday) [reply comment/tests.py]:
```
@mock.patch('utils.send_comment_to_admin_email')
def test_should_send_mail_to_admin(self, mocked_send_email):
        ...
        mocked_send_email.assert_called_once()
```

File: comment/utils.py [hunk 6]
```diff
@@ -1,10 +1,11 @@
 from django.conf import settings
+from django.db.models import Q
 from django.core.mail import send_mail
 from django.contrib.auth.models import User
 
 
 def send_comment_to_admin_email(comment):
-    admin = User.objects.filter(is_superuser=True)[0]
+    admin = User.objects.filter(~Q(email=''), is_superuser=True).first()
```
Comment (dadaday) [inline comment/utils.py:8]: Should it work like this? Admin that receives the emails will always be superuser?

File: comment/utils.py [hunk 7]
```diff
@@ -1,10 +1,11 @@
 from django.conf import settings
+from django.db.models import Q
 from django.core.mail import send_mail
 from django.contrib.auth.models import User
 
 
 def send_comment_to_admin_email(comment):
```
Comment (dadaday) [inline comment/utils.py]: It is a bit unclear what does `comment` parameter contain. Of course, from `CommentViews` we know that it contains the dict with serialized data, however it is not straightforward. 

As an option you can pass parameters from `CommentViews` like this:
```
utils.send_comment_to_admin_email(name=serializer.validated_data['name'], phone=serializer.validated_data['phone'], ...)
```
The tradeoff is that the function will have a lot of parameters which may have own disadvantages :)

**Review by dadaday — CHANGES_REQUESTED** [review]

File: aboutus/tests.py (hunk 1 above)
Comment (Rysbai) [reply aboutus/tests.py:17]: No, but I think I should use.

File: aboutus/tests.py (hunk 2 above)
Comment (Rysbai) [reply aboutus/tests.py]: Mm, OK. Thanks.

File: aboutus/tests.py (hunk 3 above)
Comment (Rysbai) [reply aboutus/tests.py]: OK.

File: aboutus/tests.py (hunk 4 above)
Comment (Rysbai) [reply aboutus/tests.py]: OK.

File: comment/tests.py (hunk 5 above)
Comment (Rysbai) [reply comment/tests.py]: Wow, cool! I didn't know about it.

**Review by dadaday — APPROVED** [review]

**Review by dadaday — APPROVED** [review]

**Pull request merged**